src/store.js

```diff
@@ -92,6 +92,9 @@ const useStoreImpl = create((set, get) => {
       honk: false,
       boost: false,
     },
+    nitro: {
```

Comment:
why is this an obj?

Reply:
True, I thought to extend with an other property and at the end was not necessary. I forgot to revert it to a single prop. I will edit. Thanks for the hint.